source/includes/plugin_info/_000-describe.md.erb

```diff
@@ -44,7 +43,11 @@ end
       version: "1.3.3",
       target_go_version: "15.1.0",
       description: "Plugin that polls a GitHub repository for pull requests and triggers a build for each of them",
-      target_operating_systems: []
+      target_operating_systems: [],
+      vendor_json:  {
```

Comment:
Typo? It's supposed to be `vendor` not `vendor_json`.

Reply:
Good catch, sorry for that! :)